app/features/icons/components/JitsiGlyph.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React, { Component } from 'react';
+
+/**
+ * Jitsi Icon component.
+ */
+class JitsiGlyph extends Component<*> {
+    /**
+     * Render function of component.
+     *
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            /* eslint-disable */
+            <svg width='24' height='24' viewBox='0 0 24 24' focusable='false' role='presentation'>
```

Comment:
can't we use some loader for this? instead of embedding the SVG in render(), that is.

Reply:
I think it might be possible with [svg-inline-loader](https://github.com/webpack-contrib/svg-inline-loader). I'll give it a try. :smile: